pingpong/src/main/java/com/cedrus/pingpong/controller/PingPongController.java

```diff
@@ -1,31 +1,48 @@
 package com.cedrus.pingpong.controller;
 
-import com.cedrus.pingpong.kafka.PingPongProducer;
+import com.cedrus.pingpong.config.TopicConfig;
 import com.cedrus.pingpong.model.PingPongMessage;
 import com.cedrus.pingpong.model.PingPongRequest;
+import com.cedrus.pingpong.model.PingPongResponse;
+import com.cedrus.pingpong.service.StartGameService;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class PingPongController {
-    @Autowired private PingPongProducer producer;
-
-    @RequestMapping(value = "/ball", method = RequestMethod.POST)
-    @ResponseBody
-    public String startPing(@RequestBody PingPongRequest pingPongRequest) throws JsonProcessingException {
```

Comment:
Please return a complex object, wrapped in ResponseEntity

Reply:
This is the master branch's current code, the Doubles branch uses a ResponseEntity